requirements/edx/custom.txt

```diff
@@ -1,36 +1,36 @@
-# FIXME: temp hash, until (https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/207 is merged into `master`)
-#-e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2.git@77dde12fc405ca65964f32ad93849504aa53ad8d#egg=xblock-drag-and-drop-v2==2.2.9
+-e git+https://github.com/edx-solutions/xblock-mentoring.git@rebase-juniper#egg=xblock-mentoring==1.0
+-e git+https://github.com/edx-solutions/xblock-image-explorer.git@juniper-rebase#egg=xblock-image-explorer==2.0
+-e git+https://github.com/edx-solutions/xblock-drag-and-drop.git@rebase-juniper#egg=xblock-drag-and-drop==1.0
+-e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2.git@apros-master#egg=xblock-drag-and-drop-v2==2.5.2
 # This is required for A2E courses that were created with the temporary (xblock-drag-and-drop-v2-new) DnDv2 branch to continue to work.
 # FIXME: bump version to 2.1.6 when https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/154 merged and tagged
 #-e git+https://github.com/open-craft/xblock-drag-and-drop-v2.git@82c9dc5e16d10793e8b79e60661e1a78893fce25#egg=xblock-drag-and-drop-v2-new
-#-e git+https://github.com/edx-solutions/xblock-ooyala.git@v4.0.5#egg=xblock-ooyala==4.0.5
+-e git+https://github.com/edx-solutions/xblock-ooyala.git@rebase-juniper#egg=xblock-ooyala==5.0.0
+
 #git+https://github.com/edx-solutions/xblock-group-project.git@0.1.3#egg=xblock-group-project==0.1.3
-#-e git+https://github.com/edx-solutions/xblock-adventure.git@0.2#egg=xblock-adventure==0.2
-#-e git+https://github.com/open-craft/xblock-poll.git@v1.9.10#egg=xblock-poll==1.9.10
-#-e git+https://github.com/open-craft/problem-builder.git@v3.4.22#egg=xblock-problem-builder==3.4.22
-#-e git+https://github.com/OfficeDev/xblock-officemix.git@86238f5968a08db005717dbddc346808f1ed3716#egg=xblock-officemix
-#-e git+https://github.com/open-craft/xblock-chat.git@v0.2.8#egg=xblock-chat==0.2.8
-#-e git+https://github.com/open-craft/xblock-eoc-journal.git@v0.6#egg=xblock-eoc-journal==0.6
-#-e git+https://github.com/mckinseyacademy/xblock-scorm.git@v2.2#egg=xblock-scorm==2.2
-#-e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@v0.2.5#egg=xblock-diagnostic-feedback==0.2.5
+-e git+https://github.com/edx-solutions/xblock-adventure.git@rebase-juniper#egg=xblock-adventure==0.3
+-e git+https://github.com/open-craft/xblock-poll.git@master#egg=xblock-poll==1.10.1
+-e git+https://github.com/open-craft/problem-builder.git@master#egg=xblock-problem-builder==4.1.6
+#-e git+https://github.com/OfficeDev/xblock-officemix.git@86238f5968a08db005717dbddc1.9.6346808f1ed3716#egg=xblock-officemix
+-e git+https://github.com/open-craft/xblock-chat.git@master#egg=chat-xblock==0.3.0
+-e git+https://github.com/open-craft/xblock-eoc-journal.git@master#egg=xblock-eoc-journal==0.9.1
+-e git+https://github.com/mckinseyacademy/xblock-scorm.git@master#egg=xblock-scorm==3.0
+-e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@master#egg=xblock-diagnostic-feedback==0.3.0
```

Comment:
@attiyaIshaque Do we need to use master branches for these? Please use the version tags instead of branch name for all repos which have been merged to master branches. Also please make sure the release tags have been created before updating versions.

Reply:
@ihtram I have addressed your comment.